lib/card.rb

```diff
@@ -76,6 +61,12 @@ def checklists
     end
   end
 
+  def filtered_checklists
+    checklists.reject do |checklist|
+      @settings.no_task_checklists.include?(checklist.name)
```

Comment:
@bgeuken as @jimmykarily said [in his comment](https://github.com/openSUSE/trollolo/pull/47/files#r90401530) this will break if `@settings` is `nil`, which you used as default [here](https://github.com/openSUSE/trollolo/pull/47/files#diff-11fdc3f293c56c46a28349b86bba1764R2)

And I also agree with him on this:

> I think you can easily add a default value of nil in all methods